draft-ietf-rtcweb-security-arch.xml

```diff
@@ -1722,12 +1722,25 @@ a=sendrecv
                   </list>
                 </t>
                 <t>
-                  Sites that have identities that do not fit into the RFC822
-                  style (for instance, identifiers that are simple numeric
-                  values, or values that contain '@' characters) SHOULD convert
-                  them to this form by escaping illegal characters and appending
-                  their IdP domain (e.g., user%40133@identity.example.com), thus
-                  ensuring that they are authoritative for the identity.
+                  Sites that use identities that are not of the form
+                  &lt;user&gt;@&lt;domain&gt;, or which use identities in
+                  which the domain portion does not match the domain name of
```

Comment:
I think this is a new requirement. It means that an IDP proxy of "identities-r-us.com" cannot serve identities of the form "user@federated-domain.com" without using the form "user%40federated-domain@identities-r-us.com".
It would be more aesthetically pleasing, at least to me, if the identity of the identity provider was carried in a different field than the identity of the user.

Reply:
That's already true, because the text immediately above this says that if the identity doesn't match the IdP proxy address, then it must be rejected. This text simply tells IdP proxies how not to have their identities rejected on that basis.